kemitix/trello_client_dart
# Clean code and rewrite CLI tests to use cliTest"

- Standardise option and flag specification
- member_get_command: add fields option
- Add --fields option to select fields returned
- convert board cli tests to use cliTest
- add card_test
- Convert card_attachment_download_command_test to cliTest
- Remove fp_nullx and fp_task_either
- Remove fp_types
- Remove unused functions in fp_either and fp_reader
- Move external out of src
- Convert card_attachment_get_command_test to use cliTest
- Rearrange cliTest parameters to group expectations
- Add tests for missing parameters
- Convert card_get_command_test to use cliTest
- Convert card_member_add_command_test to use cliTest
- Convert card_member_remove_command_test to use cliTest
- Rename cliTest parameter expected as expectations
- Convert card_update_command_test to use cliTest
- Convert list_cards_list_command_test to use cliTest
- Convert member_boards_list_command_test to use cliTest
- Convert member_get_command_test to use cliTest
